packages/verser/src/lib/verser.ts

```diff
@@ -23,9 +24,12 @@ export class Verser extends TypedEmitter<Events> {
         super();
         this.server = server;
 
-        this.server.on("connect", (req, socket) => {
+        this.server.on("connect", (req, socket: Socket) => {
             this.logger.info("New connection:", req.url);
 
+            socket.setTimeout(0);
```

Comment:
As I understand timeout=0 is just for testing, not as permanent solution?

Reply:
permanent, https://nodejs.org/api/net.html#socketsettimeouttimeout-callback

Reply:
Ah now I see, thx for claryfying :)